### DISCUSSION TITLE
Adding Award Schemes

### DISCUSSION THREAD

**Discussion by creativetux** [magicbug/Cloudlog#3012] [Q&A]
Hi Folks,  I've noticed quite alot of activity around the Bunkers on the Air programme.  Is there an easy way to add new Awards to the database or are they all hard coded?
Moving from Log4OM I could add new award schemes when they popped up and just import all the references from a CSV or create them on the fly.  Is this something available in Cloudlog?  I can't see anything relating to adding Awards in the frontend UI.

I did have a search and noticed an old post from 2020 days, but notice POTA has been added somehow.

Cheers

**Comment by magicbug** [comment]
Awards are all custom built, as Bunkers doesn't have a recognised ADIF tag it would use the SIG fields in Cloudlog.
